app/views/content_items/consultation.html.erb

```diff
@@ -0,0 +1,232 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, @content_item.title %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: t("content_item.format.#{@content_item.document_type}", count: 1),
+        title: @content_item.title,
+        average_title_length: 'long' %>
+  </div>
+</div>
+
+<%= render 'shared/withdrawal_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/metadata',
+        from: @content_item.from,
+        part_of: @content_item.part_of,
+        first_published: @content_item.published,
+        last_updated: @content_item.updated,
+        other: {
+          'Applies to' => @content_item.applies_to
+        },
+        see_updates_link: true
+    %>
+  </div>
+</div>
+<%= render 'shared/history_notice', content_item: @content_item %>
+
+<% if @content_item.unopened? %>
+  <div class="consultation-notice">
+    <h2>
+      This consultation opens <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %> <time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time>
+    </h2>
+  </div>
+<% elsif @content_item.pending_final_outcome? %>
+  <div class="consultation-notice">
+    <h2>
+      We are analysing your feedback
+    </h2>
+    <p>Visit this page again soon to download the outcome to this public&nbsp;feedback.</p>
+  </div>
+<% elsif @content_item.final_outcome? %>
+  <div class="consultation-notice">
+    <h2>
+      This consultation has concluded
+    </h2>
+  </div>
+
+  <% if @content_item.final_outcome_documents? %>
+    <div class="grid-row sidebar-with-body">
+      <div class="column-third">
+        <h1 class="section-title" id="final-outcome-documents-title">
+          Download the full outcome
+        </h1>
+      </div>
+      <div class="column-two-thirds" aria-labelledby="final-outcome-documents-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.final_outcome_documents,
+            direction: page_text_direction %>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="grid-row sidebar-with-body">
+    <div class="column-third">
+      <h1 class="section-title" id="final-outcome-detail-title">
+        <%= t("consultation.final_outcome_detail") %>
```

Comment:
We're not consistently translating strings in this file. Should for example "Download the full outcome", "This consultation has concluded" also be translated?